docs/_includes/common/field.html

```diff
@@ -502,6 +502,49 @@ <h4>Text on Side of Field</h4>
 </span>
     {% endhighlight %}
 
+    <p>To change the type of decription you would use <span class="highlight">field__description--confirmation</span>, <span class="highlight">field__description--information</span>, or <span class="highlight">field__description--attention</span>.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="field" style="margin-top: 0">
+                <label class="field__label field__label--stacked" for="field-description-variant-1">Field 1</label>
+                <span class="textbox">
+                    <input aria-describedby="field-descripiton-variant-1-description" class="textbox__control" id="field-description-variant-1" type="text" placeholder="placeholder text" />
+                </span>
+                <span class="field__description" id="field-descripiton-variant-1-description">
+                    <span>Field description - default</span>
+                </span>
+            </div>
+            <div class="field" style="margin-top: 0">
+                <label class="field__label field__label--stacked" for="field-description-variant-2">Field 2</label>
+                <span class="textbox">
+                    <input aria-describedby="field-descripiton-variant-2-description" class="textbox__control" id="field-description-variant-2" type="text" placeholder="placeholder text" />
+                </span>
+                <span class="field__description field__description--confirmation" id="field-descripiton-variant-2-description">
+                    <span>Field description - confirmation</span>
+                </span>
+            </div>
+            <div class="field" style="margin-top: 0">
```

Comment:
Minor, but I don't believe the `margin-top: 0` is necessary. Field margins should be collapsed automatically.

Reply:
Oh, this was copy/pasta. Let me remove.